lib/common/monitoring/Monitoring.ts

```diff
@@ -101,4 +105,16 @@ export abstract class Monitoring implements IDashboardSegment {
    * Returns widgets to be placed on the main dashboard.
    */
   abstract widgets(): IWidget[];
+
+  widgetsForDashboard(name: string): IWidget[] {
+    if (name === DefaultDashboards.SUMMARY) {
```

Comment:
I actually kind of prefer `switch` blocks for these sort of things.

Reply:
Fixed.